phsyn-t3/cod-u8776t_phsyn-t3.xml

```diff
@@ -57,881 +57,1053 @@
-            ‘non-risibile’ convertuntur. Et ideo necessario sequitur ‘si Sortes non videt se
-            videntem se, ergo non videt se’, per locum a convertibili. Ergo a primo: ‘Si Sortes
-            non videt aliquem hominem ‘videntem se, non videt se’. Et hec est
-            alia pars contradictionis. Ex qua cum aliera expositiva prime sequitur reliqua
-            pars contradictionis. Et hoc modo Sortes videt omnem hominem non videntem se et
-            ipse non26 videt se. Ergo videt se. Et sic videt se et non videt se. Hic autem est
-            contradictio. Et sequitur ex duabus expositivis prime. Et ille expositive intelliguntur
-            in prima. Et quia contradictio intelligitur in prima. Et sic claudit intra se
+            intra se utramque partem contradictionis. Et
+            <lb ed="#T"/> sic sequitur pars ad totum, sicut hic: ‘Sortes currit et non currit; ergo Sortes currit’.
+            <lb ed="#T"/> Et’ est locus a toto integrali; contradictio enim integratur ex
+            <lb ed="#T"/> suis partibus. Unde posita contra dictione necesse est ex ea sequi tam
+            <lb ed="#T"/> affirmationem quam negationem, per locum a toto integrali.</p>
+
+          <p xml:id="phsyn-t3-d1e362"> Item.
+            <lb ed="#T"/> Queritur de hoc sophismate POSSIBILE EST SORTEM VIDERE <add place="above-line">TANTUM</add> OMNEM HOMINEM
+            <lb ed="#T"/> NON VIDENTEM SE. Probatio. Possibile est Sortem videre <add place="above-line">tantum</add> omnem hominem
+            <lb ed="#T"/> cecum. Sed omnis homo cecus est homo non videns se. Ergo possibile est
+            <lb ed="#T"/> Sortem videre <add place="above-line">tantum</add> omnem hominem non videntem. <pc type="#pilcrow"/>Contra. Si possibile est,
+            <lb ed="#T"/> ponatur: Sortes videt <add place="above-line">tantum</add> omnem hominem videntem se. Tunc aut ´
+            <lb ed="#T"/> Sortes videt se aut non videt se. Si non videt se, ergo est
+            <lb ed="#T"/> homo non videns se. Et videt quemquam talem. Ergo videt se.
+            <lb ed="#T"/> Si videt se, ergo est homo videns se. Sed nullum talem
+            vide<lb ed="#T"/>bat. Ergo non videt, <add place="above-line">se</add> quoniam tantum videbat non videntem se.</p>
+
+          <p xml:id="phsyn-t3-d1e366"><pc type="#pilcrow"/>Solutio. Prima est impossibilis, hec scilicet ‘ possibile est Sortem videre tantum omnem
+            <lb ed="#T"/> hominem non videntem Se’. Et ista similiter ‘Sortes videt tantum omnem
+            <lb ed="#T"/> hominem non videntem se <!--there is a marginalia here: "et omnis propositio implicans contradictione est impossibilis propositio"!-->, quia utraque habet in se
+            contradictionem, Ut iam
+            <lb ed="#T"/> patebit. Probatio autem peccat secundum
+            <seg type="correction">consequens
+            <subst>
+              <del></del>
+              <add>con</add>
+            </subst>
+          </seg> ab inferiori ad
```

Comment:
Something doesn't seem correct here. 

I imagine you mean: <seg type="correction"><add>con</add>sequens</seg>` 

If there is just an addition and no deletiong you do not need to use `<subst>` and again `<seg type="correcion">` should surround the entire word where the correction is occurring.